workspace.json

```diff
@@ -3405,6 +3436,126 @@
         }
       }
     },
+    "icelandic-names-registry-backend": {
```

Comment:
Guessing you want to build a docker image for this application at some point. You'll need to add a `docker-express` empty section to trigger docker building. I'm fine with doing that as a separate task, just pointing it out.